gobblin-data-management/src/main/java/gobblin/data/management/copy/hive/HiveUtils.java

```diff
@@ -83,16 +84,20 @@
    *               (e.g. "part = \"part1\"" or "date > \"2015\"".
    * @return a list of {@link Partition}s
    */
-  public static List<Partition> getPartitions(IMetaStoreClient client, Table table, Optional<String> filter)
+  public static List<Partition> getPartitions(IMetaStoreClient client, Table table,
+      Optional<String> filter, Optional<String> timeGranularity)
```

Comment:
Also, instead of just modifying the method, add a new backwards compatible method:
```java
public static List<Partition> getPartitions(IMetaStoreClient client, Table table,
      Optional<String> filter) {
   return getPartitions(client, table, filter, Optional.absent());
}
```